src/bottom-bar/main-tool-bar.js

```diff
@@ -81,23 +82,23 @@ export default function MainToolBar() {
                 <CompleteButton disabled={locked} />
             </div>
             <ErrorSummary />
+            <FormExpiryInfo />
             {data?.completeStatus?.lastUpdatedBy && (
                 <span
                     className={cx(styles.completionSummary, styles.toolbarItem)}
                 >
-                    <span
-                        className={cx(styles.icon, styles.goToInvalidValueIcon)}
-                    >
+                    <span className={cx(styles.icon)}>
                         <IconInfo16 color={colors.grey700} />
                     </span>
 
                     <span>
-                        <span className={styles.completedByLabel}>
-                            {data.completeStatus.complete
-                                ? i18n.t('Last completed by')
-                                : i18n.t('Last incompleted by')}
-                        </span>
-                        {data?.completeStatus.lastUpdatedBy}
+                        {data.completeStatus.complete
+                            ? i18n.t('Last completed by {{-username}}', {
+                                  username: data?.completeStatus.lastUpdatedBy,
+                              })
+                            : i18n.t('Last incompleted by {{-username}}', {
+                                  username: data?.completeStatus.lastUpdatedBy,
+                              })}
```

Comment:
I added the username to the string here for i18n because I think the username might not end up in the last position in every translation -- this freed up an extra span and CSS class

Reply:
Agree on the translation point, but I'm not sure the i18next scanner handles optional chaining well?

I get this error when starting:
```
Generating internationalization strings... 
i18next-scanner: Unable to parse code "{
                                  username: data?.completeStatus.lastUpdatedBy,
                              }"
```

I think I remember the same issue when working on the data-exchange-app, and I just defined the interpolation value outside of the i18n translation. I think this might be resolved in more recent versions of i18next-scanner, so it could also be an issue of updating cli-app-scripts?

Reply:
To build on what @tomzemp said, I think it's important to consider whether or not we actually needed conditional chaining here in the first place. And in this case we didn't because if we consider the surrounding code we find that this part is only going to be rendered when `data?.completeStatus?.lastUpdatedBy` is _truthy_: 
https://github.com/dhis2/aggregate-data-entry-app/blob/8a7c0633adcf1c10ab2239769cfd4c0a578ff4e7/src/bottom-bar/main-tool-bar.js#L86-L104

So in this case we can simply remove the conditional chaining operator `?`.

BTW this problem was introduced prior to the current PR and in my opinion is a great illustration of why, in general, we should only apply conditional chaining (or conditional log in general really) when we know it is needed. Adding `?.`, `||` and `&&` without a specific case in mind might make the code look very robust, but it is a recipe for disaster in the long run. See this Wikipedia page about [fail-fast systems](https://en.wikipedia.org/wiki/Fail-fast#:~:text=In%20systems%20design%2C%20a%20fail,continue%20a%20possibly%20flawed%20process.) for a more detailed explanation.

Reply:
Ah yeah, thanks to both for pointing this out -- I think removing the conditional chaining for the reason Hendrik described is good. Good point about fail-fast systems too 👍